### PR TITLE
Migrated to the brand new heroku server.

### PR DESCRIPTION
**Pull request by genaforvena**
Changed app icon.

### CODE REVIEW

**Comment by genaforvena** [comment]
fixes #92

**Comment by genaforvena** [comment]
@oleg-popenov 
@sanovikov71

File: app/src/main/java/org/imozerov/streetartview/storage/model/Extensions.kt [hunk 1]
```diff
@@ -17,7 +17,6 @@ fun RealmArtObject.copyDataFromJson(artwork: Artwork) {
             val realmAuthor = RealmAuthor()
             realmAuthor.id = it.id
             realmAuthor.name = it.name
-            realmAuthor.photo = it.photo
```
Comment (oleg-popenov) [inline app/src/main/java/org/imozerov/streetartview/storage/model/Extensions.kt:20]: Do we no longer need author's photo? If so it gonna be replaced from server too.

File: app/src/main/java/org/imozerov/streetartview/ui/extensions/UiExtensions.kt [hunk 2]
```diff
@@ -16,8 +16,8 @@ import org.imozerov.streetartview.R
  */
 val TAG = "UiExtensions"
 
-fun ImageView.loadImage(imagePath: String) {
-    if (imagePath.isNotBlank()) {
+fun ImageView.loadImage(imagePath: String?) {
+    if (imagePath?.isNotBlank() == true) {
```
Comment (oleg-popenov) [inline app/src/main/java/org/imozerov/streetartview/ui/extensions/UiExtensions.kt:20]: Just stupid question... :flushed: 
Does `imagePath?.isNotBlank()` return `null` when `imagePath` is `null`? That's why do we need to check `== true`?

File: app/src/main/java/org/imozerov/streetartview/storage/model/Extensions.kt (hunk 1 above)
Comment (genaforvena) [inline app/src/main/java/org/imozerov/streetartview/storage/model/Extensions.kt:20]: I think we don't need it.

File: app/src/main/java/org/imozerov/streetartview/ui/extensions/UiExtensions.kt (hunk 2 above)
Comment (genaforvena) [inline app/src/main/java/org/imozerov/streetartview/ui/extensions/UiExtensions.kt:20]: imagePath?.isNotBlank() returns true? so we have to unwrap it.

**Pull request merged**